app/models/complete_moab.rb

```diff
@@ -26,14 +26,6 @@ class CompleteMoab < ApplicationRecord
   # NOTE: size here is approximate and not used for fixity checking
   validates :size, numericality: { only_integer: true, greater_than: 0 }, allow_nil: true
 
-  scope :by_moab_storage_root_name, lambda { |name|
```

Comment:
nice - also changed it in `lib/audit/checksum.rb` 👍

Reply:
Right, same pattern: `OtherModel.find_by!(...).association`